tests/test_aliases.py

```diff
@@ -106,12 +119,19 @@ class Model(BaseModel):
     with pytest.raises(ValidationError) as exc_info:
         Model(lastUpdatedBy='foo', last_updated_by='bar')
     assert exc_info.value.errors() == [
-        {'loc': ('last_updated_by',), 'msg': 'extra fields not permitted', 'type': 'value_error.extra'}
+        {
+            'input': 'bar',
+            'loc': ('last_updated_by',),
+            'msg': 'Extra inputs are not permitted',
+            'type': 'extra_forbidden',
+        }
     ]
 
 
 @pytest.mark.xfail(reason='working on V2')
 def test_alias_child_precedence():
+    # TODO: Need to decide what we are doing with fields in model_config
```

Comment:
Question 2: Re-confirming this: Are we sure we want to drop `model_config['fields']`/`Config.fields`?

In v1, it seems you can do arbitrary updates to the `FieldInfo`s for the fields by setting this attribute; there are a good number of tests using this to set JSON schema data, and using it to affect aliases, especially for fields defined in parent classes.

I think if we want to drop this, we need to either describe to users how they should override the field info from fields defined in parent classes (which I'm not sure how to do now), or we need to tell people that we no longer support that (which I think _may_ be a pain point for anyone who is currently using that functionality).